deploy/helm/templates/metrics-deployment.yaml

```diff
@@ -60,11 +60,11 @@ spec:
                 values:
                 - linux
       {{- end }}
-      {{- if or (and .Values.otel.metrics.prometheus_check .Values.otel.metrics.prometheus.url) .Values.otel.metrics.swi_endpoint_check }}
       {{- if .Values.imagePullSecrets }}
       imagePullSecrets:
```

Comment:
I'm not sure why `imagePullSecrets` was under the init containers condition and not separate, as it is in other deployments.